src/jsset.h

```diff
@@ -0,0 +1,30 @@
+/**
+* Unless explicitly stated otherwise all files in this repository are licensed under the Apache-2.0 License.
+* This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+**/
+
+#ifndef SRC_JSSET_H_
+#define SRC_JSSET_H_
+
+#include <napi.h>
+class JsSet : public Napi::Object {
```

Comment:
do you need to use a `Set` from the js side? wouldn't be faster to use a c++ set?

Reply:
IIRC, the c++ object instance for `Napi::Value` in not necessarily the same for the same instance of nodejs object. If we use the js we can trust that it will work as expected.

Reply:
oh sounds a good reason :)

also i searched for a `Napi::Set` but it seems it doesn't exists...

Reply:
oh sounds a good reason :)

also i searched for a `Napi::Set` but it seems it doesn't exists...